server/backends/blobstore/blobstore.go

```diff
@@ -358,11 +373,13 @@ func NewAwsS3BlobStore(awsConfig *config.AwsS3Config) (*AwsS3BlobStore, error) {
 			return nil, err
 		}
 	}
+	log.Debug("AWS blobstore configured")
 	return awsBlobStore, nil
 }
 
 func (a *AwsS3BlobStore) createBucketIfNotExists(ctx context.Context, bucketName string) error {
 	// HeadBucket call will return 404 or 403
+	log.Debug("Checking if AWS blobstore bucket exists")
```

Comment:
Do you want to log if it exists or not in the HeadBucketWithContext and below?

EDIT: also log the bucketName

log.Debugf("Checking if AWS blobstore bucket %q exists", bucketName)